DNN Platform/Library/Startup.cs

```diff
@@ -28,6 +30,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<ReflectedModuleControlFactory>();
             services.AddSingleton<IDnnContext, DotNetNukeContext>();
 
+            services.AddScoped<IEventLogger, EventLogController>();

```

Comment:
```suggestion
            services.AddScoped<IEventLogger, EventLogController>();
            services.AddScoped<IEventLogConfigService, EventLogController>();
```

Reply:
Thanks for catching the missed registration, this needs to be updated.

Reply:
This has been updated in the latest code push, thanks for the feedback